### PR TITLE
[ZT] add user seat management page

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Review by kennyj42 — APPROVED** [review]

**Comment by kennyj42** [comment]
Love the opportunity to clean up the distinction between seats and sessions

File: content/cloudflare-one/identity/users/seat-management.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+pcx-content-type: how-to
+title: Seat management
+weight: 4
+---
+
+# Manage user seats
+
+When an end user authenticates to an application protected by Cloudflare Access or enrolls their device into Cloudflare Gateway, the user consumes one seat. The user continues to occupy that seat regardless of the number of applications accessed or login events. Once the total amount of seats in the subscription has been consumed, additional users who attempt to login are blocked.
```
Comment (abracchi-tw) [inline content/cloudflare-one/identity/users/seat-management.md]:
```suggestion
When an end user authenticates to an application protected by Cloudflare Access or enrolls their device into Cloudflare Gateway, the user consumes one seat. The user continues to occupy that seat regardless of the number of applications accessed or login events. Once the total amount of seats in the subscription has been consumed, additional users who attempt to log in are blocked.
```

File: content/cloudflare-one/identity/users/session-management.md [hunk 2]
```diff
@@ -29,117 +19,127 @@ When users log in to an application protected by Access, Access generates two se
...
-- **Lax** - Cookies are allowed to be sent with top-level navigations and will be sent along with GET requests initiated by third party websites.
-- **Strict** - Cookies will only be sent in a first-party context and not be sent along with requests initiated by third party websites.
+Do not enable HttpOnly if:
 
-See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite) for specific behaviors of these options.
+- You are using the Access application for non-browser based tools.
+- You have software that relies on being able to access a user’s cookie generated by Access.
 
-#### When not to use SameSite
+### Enable Binding Cookie
```
Comment (abracchi-tw) [inline content/cloudflare-one/identity/users/session-management.md]:
```suggestion
### Enable the Binding Cookie
```
Also do we treat `Binding Cookie` as a proper noun?

File: content/cloudflare-one/identity/users/session-management.md [hunk 3]
```diff
@@ -29,117 +19,127 @@ When users log in to an application protected by Access, Access generates two se
...
+#### When not to use SameSite
 
-- Do not use the Binding Cookie for non-browser based Access applications that rely on protocols like SSH, RDP, etc.
+Do not enable SameSite restrictions if you have additional sites or applications that rely on a specific application’s authorization cookie.
 
-### SameSite Attribute
+### HttpOnly
 
-The [SameSite](https://web.dev/samesite-cookies-explained/) Attribute selector is a cookie attribute that restricts the cookie to only being sent if the cookie’s defined site matches the site being requested in the browser. This adds protection against CSRF attacks.
+The HttpOnly flag is a cookie attribute that prevents the cookie from being accessed by any client-side scripts, reducing the likelihood of XSS attacks. This flag is enabled by default.
```
Comment (abracchi-tw) [inline content/cloudflare-one/identity/users/session-management.md]: Should we specify `Cross-Site Scripting (XSS)`?

**Review by abracchi-tw — APPROVED** [review]

File: content/cloudflare-one/identity/users/session-management.md (hunk 3 above)
Comment (ranbel) [inline content/cloudflare-one/identity/users/session-management.md]:
```suggestion
The HttpOnly flag is a cookie attribute that prevents the cookie from being accessed by any client-side scripts, reducing the likelihood of Cross-Site Scripting (XSS) attacks. This flag is enabled by default.
```

File: content/cloudflare-one/identity/users/session-management.md (hunk 2 above)
Comment (ranbel) [reply content/cloudflare-one/identity/users/session-management.md]: Yeah, Binding Cookie is capitalized throughout this page.

File: content/cloudflare-one/identity/users/session-management.md [hunk 4]
```diff
@@ -29,117 +19,127 @@ When users log in to an application protected by Access, Access generates two se
...
-- **Lax** - Cookies are allowed to be sent with top-level navigations and will be sent along with GET requests initiated by third party websites.
-- **Strict** - Cookies will only be sent in a first-party context and not be sent along with requests initiated by third party websites.
+Do not enable HttpOnly if:
 
-See [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite) for specific behaviors of these options.
+- You are using the Access application for non-browser based tools.
+- You have software that relies on being able to access a user’s cookie generated by Access.
 
-#### When not to use SameSite
+### Enable Binding Cookie
```
Comment (ranbel) [inline content/cloudflare-one/identity/users/session-management.md]:
```suggestion
### Binding Cookie
```

File: content/cloudflare-one/identity/users/session-management.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/identity/users/session-management.md]: yes, good idea

**Pull request merged**